src/forms/inputs/dropdown-checkbox-group.js

```diff
@@ -59,13 +59,14 @@ function DropdownCheckboxGroup(props) {
     input: { name, value },
     label,
   } = props
+  const inputId = `${name}-label`
   return (
-    <fieldset>
-      <InputLabel {...{ label, name }} />
+    <div>
+      <FieldsetLegend {...{ label, name }} id={inputId} />
       <DropdownSelect selectedValues={value} className="checkboxes">
-        <CheckboxGroup {...props} label={false} />
+        <CheckboxGroup {...props} label={false} ariaLabelledby={inputId} />
       </DropdownSelect>
-    </fieldset>
+    </div>
```

Comment:
The checkbox group is actually the `fieldset` so `div` should be correct here and since a label/legend isn't provided, I'm using the `id` on the "outer" legend to provide the aria labelledby.

Reply:
If we aren't using a `fieldset`, is `legend` the semantically appropriate tag? Or would `p`/`span` suffice?

Looking at the specs, the only Permitted parent is a `fieldset` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend#technical_summary